test/Core.Test/Auth/Models/Business/Tokenables/RegistrationEmailVerificationTokenableTests.cs

```diff
@@ -0,0 +1,184 @@
+﻿using AutoFixture.Xunit2;
+using Bit.Core.Tokens;
+
+namespace Bit.Core.Test.Auth.Models.Business.Tokenables;
+using Bit.Core.Auth.Models.Business.Tokenables;
+using Xunit;
+
+public class RegistrationEmailVerificationTokenableTests
+{
+    // Allow a small tolerance for possible execution delays or clock precision to avoid flaky tests.
+    private static readonly TimeSpan _timeTolerance = TimeSpan.FromMilliseconds(10);
+
+    /// <summary>
+    /// Tests the default constructor behavior when passed null/default values.
+    /// </summary>
+    [Fact]
+    public void Constructor_NullInputs_PropertiesSetToDefault()
+    {
+        var token = new RegistrationEmailVerificationTokenable(null, null, default);
+
+        Assert.Equal(default, token.Name);
+        Assert.Equal(default, token.Email);
+        Assert.Equal(default, token.ReceiveMarketingEmails);
+    }
```

Comment:
If this would fail validation, should we just throw in the constructor when provided null values? This seems fine, but seems reactive.

Reply:
Good call! I added email validation to the constructor and updated the tokenable tests!